magic-modules/create-pr.sh

```diff
@@ -10,15 +10,24 @@ set -e
 set -x
 
 shopt -s dotglob
-cp -r magic-modules/* magic-modules-out
+cp -r magic-modules/* magic-modules-with-comment
 
-cd magic-modules-out
+pushd magic-modules-with-comment
 
 # This says "check out the branch which contains HEAD, and set it up to track its upstream."
 git checkout -t "$(git branch -a --contains "$(git rev-parse HEAD)" | grep -v "detached")"
 
-cd build/terraform
+pushd build/terraform
```

Comment:
where is the popd to go back to the original folder? If we don't use popd, why not simply using cd?

Reply:
Nelson requested I use `pushd`/`popd` in all cases if I was going to use relative paths, so I just converted this over.  Agreed it's not necessary right now.

Reply:
👍